examples/resources/generic-secret.yaml

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2019 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: example-secret
+type: generic
+data:
+  API_KEY: bm90aGluZ3Rvc2VlaGVyZQ==
```

Comment:
seems like its already base64 encoded ?

Reply:
It just serve as an example.

Decoded it is ‘nothingtoseehere’

Reply:
It just serve as an example.

Decoded it is ‘nothingtoseehere’